### PR TITLE
Add Porting Kit

### PR DESCRIPTION
**Pull request by sanyer**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
The conditional makes this cask a mess. You’re basically introducing two casks in one. Submit the `legacy` as its own version in [caskroom/versions](https://github.com/caskroom/homebrew-versions) instead.

**Comment by sanyer** [comment]
@vitorgalvao yeah, I agree. Didn't think, though, about the caskroom/versions. Thanks, will fix later today.

File: Casks/porting-kit.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  app 'Porting Kit.app'
+
+  zap delete: [
+                '~/Library/Preferences/edu.ufrj.vitormm.Porting-Kit.plist',
+                '~/Library/Application Support/Porting-Kit',
+                '~/Library/Saved Application State/edu.ufrj.vitormm.Porting-Kit.savedState',
+                '~/Library/Caches/edu.ufrj.vitormm.Porting-Kit',
+                '~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/edu.ufrj.vitormm.porting-kit.sfl',
+              ],
+      rmdir:  '~/Applications/Porting Kit.app'
```
Comment (jawshooah) [inline Casks/porting-kit.rb]: This isn't necessary. The `app` stanza takes care of it.

Comment (vitorgalvao) [inline Casks/porting-kit.rb]: It also wouldn’t work anyway, now that I think about it, since `rmdir` only works on empty directories.

**Pull request merged**